controllers/admin/AdminStatusesController.php

```diff
@@ -477,7 +477,7 @@ protected function renderOrderReturnsForm()
             $back = self::$currentIndex . '&token=' . $this->token;
         }
         if (!Validate::isCleanHtml($back)) {
-            die(Tools::displayError());
+            die(Tools::displayError('Provided "back" parameter is invalid.'));
```

Comment:
I'm a bit worried about this message because `$back` is a calculated variable in `function renderOrderReturnsForm` but I can't think of a better message.